frontend/src/lib/DefaultStreamlitEndpoints.ts

```diff
@@ -74,6 +74,17 @@ export class DefaultStreamlitEndpoints implements StreamlitEndpoints {
     })
   }
 
+  public async fetchCachedForwardMsg(hash: string): Promise<Uint8Array> {
+    const serverURI = this.requireServerUri()
+    const rsp = await axios.request({
```

Comment:
`ForwardMessageCache` implemented this GET using the `fetch()` API. I changed it to use axios here to conform to the other endpoint implementations.